### PR TITLE
Update CONTRIBUTING.md for Florence

### PR DESCRIPTION
**Pull request by mal0ki**
Adjusted and edited to be geared towards Florence rather than Tootsuit.

### CODE REVIEW

**Review by clarfonthey — APPROVED** [review]

**Comment by clarfonthey** [comment]
I think this is probably good enough for now. As things go along we can update more

**Comment by mal0ki** [comment]
> 
> 
> I think this is probably good enough for now. As things go along we can update more

Yeah that was exactly what I was thinking, that we consistently work with this.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,39 +1,33 @@
...
+Thank you for considering contributing to Florence Mastodon
 
 You can contribute in the following ways:
 
 - Finding and reporting bugs
 - Translating the Mastodon interface into various languages
 - Contributing code to Mastodon by fixing bugs or implementing features
 - Improving the documentation
+- Joining the conversation on our [Chat]
+- Participate in OutReach or other activities
```
Comment (TreacherousNexus) [inline CONTRIBUTING.md]: For consistency should using "Participating" and "Proof-reading"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,39 +1,33 @@
...
+- Proof-read
+- And much more
 
-If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
+If you want to contribute with monetary support you can do so via our [Open Collective](https://opencollective.com/florence-social) (which isn't publicly launched yet)
```
Comment (TreacherousNexus) [inline CONTRIBUTING.md]: If this isn't publicly-launched, should we be linking to it yet? Is there a temporary stop-gap solution or time-frame for this?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,39 +1,33 @@
 Contributing
 ============
 
-Thank you for considering contributing to Mastodon 🐘
+Thank you for considering contributing to Florence Mastodon
```
Comment (TreacherousNexus) [inline CONTRIBUTING.md]: Might be better phrased "Thank you for your interest in contributing to Florence Mastodon"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,39 +1,33 @@
...
+- And much more
 
-If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
+If you want to contribute with monetary support you can do so via our [Open Collective](https://opencollective.com/florence-social) (which isn't publicly launched yet)
 
-## Bug reports
+## Adding an Issue for Bug Report or Feature Request
 
-Bug reports and feature suggestions can be submitted to [GitHub Issues](https://github.com/tootsuite/mastodon/issues). Please make sure that you are not submitting duplicates, and that a similar report or request has not already been resolved or rejected in the past using the search function. Please also use descriptive, concise titles.
+Bug reports and feature suggestions can be submitted to [GitHub Issues](https://github.com/florence-social/mastodon-fork/issues). Do not worry about submitting duplicates, but please make a cursory search to see if any similar reports or request have already been resolved or rejected in the past using the search function. If you find a bug or feature request which matches yours you can join the conversation by adding your commentary to that issue.
```
Comment (TreacherousNexus) [inline CONTRIBUTING.md]: Typo: "Similar reports or request" should be "Similar reports or requests"

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,39 +1,33 @@
...
 
-## Translations
-
-You can submit translations via [Crowdin](https://crowdin.com/project/mastodon). They are periodically merged into the codebase.
-
-[![Crowdin](https://d322cqt584bo4o.cloudfront.net/mastodon/localized.svg)][crowdin]
-
-## Pull requests
+Try to describe your issue, the use-case, and additional considerations when submitting a feature request.
```
Comment (TreacherousNexus) [inline CONTRIBUTING.md]: "and any additional considerations" might be better. Omitting "any" implies people may need to ask what is permitted.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,39 +1,33 @@
...
-
-The smaller the set of changes in the pull request is, the quicker it can be reviewed and merged. Splitting tasks into multiple smaller pull requests is often preferable.
-
-**Pull requests that do not pass automated checks may not be reviewed**. In particular, you need to keep in mind:
+## Translations
 
-- Unit and integration tests (rspec, jest)
-- Code style rules (rubocop, eslint)
-- Normalization of locale files (i18n-tasks)
+We will be using Weblate for translations in the near future, but we're waiting for a free account for Open Source Projects via Weblate themselves, in the meanwhile you can submit PRs with translations, or wait. We will update this document as soon as Weblate is available for translations, and will make public announcements elsewhere.
```
Comment (TreacherousNexus) [inline CONTRIBUTING.md]: Might be better rearranged to establish the supported behaviour before the future expected system, particularly if there is not an established timeframe for that yet.

eg. "You can submit pull requests for translations. In future, we plan to use Weblate and will update this document when that system is available"

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,39 +1,33 @@
...
 
 ## Documentation
 
-The [Mastodon documentation](https://docs.joinmastodon.org) is a statically generated site. You can [submit merge requests to mastodon/docs](https://source.joinmastodon.org/mastodon/docs).
+Documentation work will be added soon, and in the meanwhile you can join us in our [Chat] for documentation.
```
Comment (TreacherousNexus) [inline CONTRIBUTING.md]: "A documentation site will be made available for future versions of Florence Mastodon. In the meantime, if you have any questions you can join us on our [chat] server"

**Review by ghost — CHANGES_REQUESTED** [review]
I have added some points for discussion and minor changes. I hope this helps!

Edit: Removed comments after discussion on chat clarifying that this is just a temporary change for now in the initial release. It will be improved in a later PR.

**Review by ghost — APPROVED** [review]

**Review by clarfonthey — APPROVED** [review]

**Review by jhaye — APPROVED** [review]

**Review by 1011X — APPROVED** [review]

**Pull request merged**